connector_node/risingwave-source-test/src/test/java/com/risingwave/connector/PostgresSourceTest.java

```diff
@@ -0,0 +1,186 @@
+package com.risingwave.connector;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+import com.risingwave.proto.ConnectorServiceProto;
+import io.grpc.Grpc;
+import io.grpc.InsecureChannelCredentials;
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.locks.Condition;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+import org.junit.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.utility.MountableFile;
+
+public class PostgresSourceTest {
+    private static final Logger logger =
+            LoggerFactory.getLogger(PostgresSourceTest.class.getName());
+
+    private static final PostgreSQLContainer<?> pg =
+            new PostgreSQLContainer<>("postgres:12.3-alpine")
+                    .withDatabaseName("test")
+                    .withUsername("postgres")
+                    .withCommand("postgres -c wal_level=logical -c max_wal_senders=10");
+
+    public static Server connectorServer =
+            ServerBuilder.forPort(ConnectorService.DEFAULT_PORT)
+                    .addService(new ConnectorServiceImpl())
+                    .build();
+
+    public static SourceTestClient testClient =
+            new SourceTestClient(
+                    Grpc.newChannelBuilder(
+                                    "localhost:" + ConnectorService.DEFAULT_PORT,
+                                    InsecureChannelCredentials.create())
+                            .build());
+
+    @BeforeClass
+    public static void init() {
+        // generate orders.tbl test data
+        SourceTestClient.genOrdersTable(10000);
+        // start connector server and postgres...
+        try {
+            connectorServer.start();
+            logger.info("connector service started");
+            pg.withCopyFileToContainer(
+                    MountableFile.forClasspathResource("orders.tbl"), "/home/orders.tbl");
+            pg.start();
+            pg.withUsername("postgres")
+                    .execInContainer(
+                            "sh",
+                            "-c",
+                            "echo 'host replication postgres 172.17.0.1/32 trust' >> /var/lib/postgresql/data/pg_hba.conf");
```

Comment:
What's the purpose of this command?

Reply:
this command is to specify the clients that are permitted to replicate with the postgres, so that debezium can work properly. 
seems like currently testcontainers has no easy way for us to modify/override `pg_hba.conf` so I decided to use thi s command as a workaround..
https://debezium.io/documentation/reference/stable/connectors/postgresql.html#postgresql-host-replication-permissions